src/main/java/no/nav/tag/finnkandidatapi/kafka/OppfølgingAvsluttetConsumer.java

```diff
@@ -1,23 +1,35 @@
 package no.nav.tag.finnkandidatapi.kafka;
 
-import lombok.AllArgsConstructor;
+import io.micrometer.core.instrument.MeterRegistry;
 import lombok.extern.slf4j.Slf4j;
 import no.nav.tag.finnkandidatapi.kandidat.KandidatService;
 import org.apache.kafka.clients.consumer.ConsumerRecord;
 import org.springframework.kafka.annotation.KafkaListener;
 import org.springframework.stereotype.Component;
 
-import java.io.IOException;
-
 import static no.nav.tag.finnkandidatapi.kafka.OppfølgingAvsluttetUtils.deserialiserMelding;
 
 @Slf4j
 @Component
-@AllArgsConstructor
 public class OppfølgingAvsluttetConsumer {
 
+    private static final String AVSLUTTET_OPPFØLGING_FEILET = "finn-kandidat.avsluttet-oppfolging.feilet";
+
     private KandidatService kandidatService;
+    @SuppressWarnings({"unused", "FieldCanBeLocal"})
```

Comment:
Hva er dette?

Reply:
For å fjerne warning om at denne variabelen ikke blir brukt... Men brukes i KafkaListener annotasjonen som string...